src/org/parosproxy/paros/extension/ExtensionHookMenu.java

```diff
@@ -331,6 +347,16 @@ public void addToolsMenuItem(ZapMenuItem menuItem) {
         addToolsMenuItemImpl(menuItem);
     }
 
+    public void addImportMenuItem(ZapMenuItem menuItem) { 
+	addImportMenuItemImpl(menuItem);
+    }
+
+    private void addImportMenuItemImpl (JMenuItem menuItem) {

```

Comment:
This method can be removed, just used by `addImportMenuItem`.

Reply:
Okay, so I should just bring the code of `addImportMenuItemImpl` inside `addImportMenuItem`.